lib/components/src/controls/Range.tsx

```diff
@@ -164,13 +164,16 @@ export const RangeControl: FC<RangeProps> = ({
   const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
     onChange(parse(event.target.value));
   };
+  const inputValue = value === null ? '' : value;
```

Comment:
I would just inline this variable.

Reply:
Okay, I can do that.  I was on the fence about whether or not to inline it.